sdk/loadtestservice/loadtesting-rest/src/models.ts

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export interface AppComponentsMap {
+  /** Azure Load Testing resource Id */
+  resourceId?: string;
+  /** [Required, if testRunId is not given] Load test unique identifier */
+  testId?: string;
+  /** [Required if testId is not given] Load test run unique identifier */
+  testRunId?: string;
+  /** AppComponent name */
+  name?: string;
+  /** AppComponents Map { resource id (Fully qualified resource Id e.g subscriptions/{subId}/resourceGroups/{rg}/providers/Microsoft.LoadTestService/loadtests/{resName}) : resource object } */
+  value: Record<string, AppComponent>;
```

Comment:
Just out of curious, should the key of value the Record<string, AppComponent>  be the same with resourceId at line 6?

Reply:
This key at line 14 is different from line 6 resource id. The key(resource id) at line 14 is the resource id of app user wants to monitor, and at line 6 we have load test resource's id.